src/EllCrv/MordellWeilRank.jl

```diff
@@ -0,0 +1,365 @@
+###############################################################################
+#
+#  Descent methods for rank computations and local solubility of quartics
```

Comment:
Here and elsewhere? Could you please write some comments in the files and/or next to functions where things come from? Otherwise it'll be tough to debug let alone improve this code down the road (hehe... see also our  MarDI goals :-) )